src/cljs/rems/spa.cljs

```diff
@@ -193,6 +193,17 @@
 
 ;;;; Pages
 
+(defn login-intro []
+  [:div
+    [:div.row.justify-content-center
```

Comment:
The indentation is not correct here.

Reply:
ok, ish, looks like my Calva "fixes" indentation